Exercise_C.js

```diff
@@ -13,6 +13,15 @@ If the number is divisible by 3 and 5 it should log Hong Kong.
 See README.md if you need help running your code.
 */
 
+let arr = []
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+  if (i % 3 == 0 && i % 5 == 0) {
+    arr.push("Hong Kong")
+  } else if (i % 3 == 0 && i % 5 != 0) {
+    arr.push("Hong");
+  } else if (i % 3 != 0 && i % 5 == 0) {
+    arr.push("Kong");
+  } else arr.push(i);
 }
+

```

Comment:
Logically this works, however if you were to put this into a JS file and try to run, you get an error.
Either wrap lines 16-27 inside a function or instead of return, console.log(arr)